### DISCUSSION TITLE
Class compliant MIDI?

### DISCUSSION THREAD

**Discussion by amowry** [hathach/tinyusb#793] [Q&A] [answered]
I'm interested in making a class-compliant MIDI device using the Adafruit MIDI library and an nrf52840 device. Is there any easy way with TinyUSB to turn off the CDC class so that only the MIDI class enumerates? Thanks!

**Comment by hathach** [comment, marked as answer]
are you using Arduino, if so you could subscribe to this issue here. https://github.com/adafruit/Adafruit_TinyUSB_ArduinoCore/issues/19

**Reply by amowry** [reply]
Okay, great, thanks very much!
